docs/common_tasks.rst

```diff
@@ -51,6 +51,42 @@ Running the tests
 
      (venv)vagrant@precise32:~/treeherder$ flake8
 
+
+Look up credentials for a project
+---------------------------------
+
+To submit data to an existing project, you need corresponding OAuth credentials, even if you're submitting to your local server.
+
+Prerequisite: you must have previously exported credentials for all data source projects. This is part of the usual Treeherder setup process.
+
+  .. code-block:: bash
+
+      > ./manage.py export_project_credentials
+
+The above command writes all project credentials to ``treeherder/etl/data/
+credentials.json`` for use by the Treeherder service.
+
+If you are implementing data submission with ``TreeherderClient`` and testing
```

Comment:
Let's link to the submitting data document inline: 

    data submission -> :doc:`data submission <submitting_data>`